2.9.0/done/result_mps.md

```diff
@@ -0,0 +1,112 @@
+- Extend `index_put` to complex types ([\#160159](https://github.com/pytorch/pytorch/pull/160159))  
+- Extend addmm to integral types ([\#160270](https://github.com/pytorch/pytorch/pull/160270))  
+- Add support for unsigned types ([\#159094](https://github.com/pytorch/pytorch/pull/159094))  
+- Add API to query GPU core count ([\#160414](https://github.com/pytorch/pytorch/pull/160414))  
+- Update `avg_pool3d` kernel to use `opmath_t` ([\#161071](https://github.com/pytorch/pytorch/pull/161071))  
+- Add slow version of `kthvalue` ([\#161817](https://github.com/pytorch/pytorch/pull/161817))  
+- Type-promote tensor-iterator common dtype ([\#160334](https://github.com/pytorch/pytorch/pull/160334))  
+- Add fused\_rms and sdpa\_mps fallback ops for AOTInductor ([\#156844](https://github.com/pytorch/pytorch/pull/156844))  
+- Implement logcumsumexp metal kernel ([\#156858](https://github.com/pytorch/pytorch/pull/156858))  
+- Migrate round unary op to Metal ([\#161712](https://github.com/pytorch/pytorch/pull/161712))  
+- Move max\_pool2d to Metal for `stride != 1` ([\#157876](https://github.com/pytorch/pytorch/pull/157876))
+
+### bug fixes
+
+- Fix batch norm incorrect gradient ([\#156867](https://github.com/pytorch/pytorch/pull/156867))  
+- Do not crash if tensor dim \> INT\_MAX ([\#158824](https://github.com/pytorch/pytorch/pull/158824))  
+- Avoid outputing zeros from `exponential_` for MPS ([\#159386](https://github.com/pytorch/pytorch/pull/159386))  
+- Fix MPS autocast for ConvTranspose3d ([\#160345](https://github.com/pytorch/pytorch/pull/160345))  
+- Fix MPS conv3d autocast bias dtype mismatch ([\#160423](https://github.com/pytorch/pytorch/pull/160423))  
+- Fix error check for torch.var on scalar ([\#160889](https://github.com/pytorch/pytorch/pull/160889))  
+- Fix index\_add for complex \+ int64 ([\#160926](https://github.com/pytorch/pytorch/pull/160926))  
+- Fix constant\_pad\_nd\_mps bug when pad is empty ([\#161149](https://github.com/pytorch/pytorch/pull/161149))  
+- Fix index\_select for scalar\_types ([\#161206](https://github.com/pytorch/pytorch/pull/161206))  
+- Fix `index_copy` for scalars ([\#161267](https://github.com/pytorch/pytorch/pull/161267))  
+- Fix index\_copy for strided indices ([\#161333](https://github.com/pytorch/pytorch/pull/161333))  
+- Fix index\_add for int64 input \+ zerodim index ([\#161511](https://github.com/pytorch/pytorch/pull/161511))  
+- Ensure that tensors are contiguous before using MPS linear kernel ([\#161641](https://github.com/pytorch/pytorch/pull/161641))  
+- Address NaNs if SDPA is called with all values masked from query ([\#157727](https://github.com/pytorch/pytorch/pull/157727))  
+- Fix invalid formatting ([\#158436](https://github.com/pytorch/pytorch/pull/158436))  
+- Update `avg_pool2d` to use Metal kernel when `ceil_mode=True` ([\#161011](https://github.com/pytorch/pytorch/pull/161011))  
```

Comment:
I would've moved it to improvements, as before this change it simply errored out